### ISSUE TITLE
[$250] mWeb - BNP - In BNP, on selecting amount, drag bubble is not shown

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#43266]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.80
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4602260
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Open site in both mweb/Android app
2. Tap fab -- submit expense
3. Enter an amount 8.74
4.Select the amount entered

## Expected Result:
In BNP, on selecting amount, drag bubble must be shown on both sides of amount entered.

## Actual Result:
In BNP, on selecting amount, drag bubble is not shown on both sides of amount in mweb like android

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/d8a7c417-2d7d-455a-8aff-a9fa103f3a62

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~015eaae788488c0a1f</li>
        <li>Upwork Job ID: 1803295394511159850</li>
        <li>Last Price Increase: 2024-06-26</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @Ollyws</details>

**Comment by izarutskaya** [comment]
We think this issue might be related to the  #vip-vsb

**Comment by JmillsExpensify** [comment]
Low priority issue though added to #wave-collect as polish.

**Comment by JmillsExpensify** [comment]
Whoops, adding the external label to get proposals

**Comment by JmillsExpensify** [comment]
Waiting on proposals

**Comment by QichenZhu** [comment]
![1000002900](https://github.com/Expensify/App/assets/57348009/71252ae6-cd49-4ef3-ae11-67d2a7154296)

Cannot reproduce on `main`.

**Comment by JmillsExpensify** [comment]
Asked QA to retest

**Comment by kavimuru** [comment]
Not reproducible

https://github.com/Expensify/App/assets/43996225/a0e68dda-8f35-49f3-bd94-cf2046539053

**Comment by JmillsExpensify** [comment]
Alright, let's close this out then!
